packages/story-editor/src/components/library/panes/pageTemplates/pageTemplatesPane.js

```diff
@@ -157,9 +157,9 @@ function PageTemplatesPane(props) {
     setNextTemplatesToFetch,
   ]);
 
-  const handleToggle = () => {
-    setShowDefaultTemplates(!showDefaultTemplates);
-    localStore.setItemByKey(LOCAL_STORAGE_KEY, !showDefaultTemplates);
+  const handleSelect = (event, menuItem) => {
+    setShowDefaultTemplates(DEFAULT === menuItem);
```

Comment:
Excellent find. Only thing I would suggest would be to not reset `showDefaultTemplates` or `localStore` if the new selection is the same as what is already selected. No need to reset them if they are just gonna be the same. Something like
```
if (showDefaultTemplates !== (DEFAULT === menuItem)) {
...}
```